File: Exercises/T3/src/java/Cargos/Vendedora.java
package Exercises.T3.src.java.Cargos;
import Exercises.T3.src.java.GestaoABC.Funcionaria;

public class Vendedora extends Funcionaria{
  private double comissao;

  public Vendedora(String nome, String CPF, double salarioBase, double comissao) {
    this.nome = nome;
    this.CPF = CPF;
    this.salarioBase = salarioBase;
    this.comissao = comissao;
  };

  public double calculaSalario() {
    return this.salarioBase + this.comissao;
  };

  @Override
  public String toString() {
    String str = String.format("Cargo: Vendedora%n");
    str += this.superToString();
    return str;
  }
}

File: Exercises/T3/src/python/Cargos/Vendedora.py
from GestaoABC.Funcionaria import Funcionaria

class Vendedora(Funcionaria):
    def __init__(self, nome: str, CPF: str, salarioBase: float, comissao: float) -> None:
        super().__init__(nome, CPF, salarioBase)
        self.comissao = comissao
    
    def calculaSalario(self) -> float:
        return self.salarioBase + self.comissao

    def __str__(self) -> str:
        return "Cargo: Vendedora\n" + super().__str__()


    